docs/monitors/collectd-systemd.md

```diff
@@ -0,0 +1,96 @@
+<!--- GENERATED BY gomplate from scripts/docs/monitor-page.md.tmpl --->
+
+# collectd/systemd
+
+This [SignalFx Smart Agent](https://github.com/signalfx/signalfx-agent) monitor collects metrics about the state of
+configured systemd services using the [collectd-systemd](https://github.com/signalfx/integrations/tree/master/collectd-systemd)
+plugin. A service is in the state that a metric represents if the metric value is 1 and not in the state if the
+metric value is 0. The name of the service is assigned to dimension `plugin_instance` of the reported metric.
+
+The collectd-systemd plugin reads the status of systemd services from host location `/var/run/dbus/system_bus_socket`.
+The host location must be mounted to the SignalFx Smart Agent container in the docker deployment of the agent. The
+agent container must run in privilege mode. Below is an excerpt of the docker run command.
+```yaml
+docker run ...\
+  --privileged \
+  -v /var/run/dbus/system_bus_socket:/var/run/dbus/system_bus_socket:ro \
+...
+```
+Below is a sample SignalFx Smart Agent YAML configuration for monitoring the state of `docker` and `ubuntu-fan`
+services. See [here](https://github.com/signalfx/signalfx-agent#configuration) for SignalFx Smart Agent
+configuration details.
+```yaml
+monitors:
+- type: collectd/systemd
+  intervalSeconds: 10
+  services:
+    - docker
+    - ubuntu-fan
+```
+Only metric `gauge.substate.running` which indicates whether a service is running or not gets reported by default.
+The other metrics must be configure explicitly (see below). Also, it is recommended that you include the extra
+dimension `metric_source` with a meaningful value in order to facilitate filtering in the SignalFx app.
+```
+monitors:
+- type: collectd/systemd
+  intervalSeconds: 10
+  services:
+    - docker
+    - ubuntu-fan
+  extraDimensions:
+    metric_source: collectd/systemd
+```
+
+
+Monitor Type: `collectd/systemd`
+
+[Monitor Source Code](https://github.com/signalfx/signalfx-agent/tree/master/internal/monitors/collectd/systemd)
+
+**Accepts Endpoints**: **Yes**
+
+**Multiple Instances Allowed**: Yes
+
+## Configuration
+
+| Config option | Required | Type | Description |
+| --- | --- | --- | --- |
+| `services` | **yes** | `list of strings` | Systemd services to report on |
+| `serviceStates` | no | `map of bools` | Systemd service states. The default state is `ActiveState` and the default metric exported is `gauge.substate.running`. Possible service states are `ActiveState`, `SubState` and `LoadState` |
```

Comment:
For the user configuration, just have three boolean flags, e.g. `sendActiveState`, `sendSubState`, etc.